common/djangoapps/student/models.py

```diff
@@ -1506,10 +1506,13 @@ def change_mode(self, mode):
         """Changes this `CourseEnrollment` record's mode to `mode`.  Saves immediately."""
         self.update_enrollment(mode=mode)
 
-    def refundable(self):
+    def refundable(self, user_already_has_certs_for=None):
         """
         For paid/verified certificates, students may receive a refund if they have
         a verified certificate and the deadline for refunds has not yet passed.
```

Comment:
I'll just align the comment with what's true in the code. I was mostly curious if there's some weird edge case in the data model, and it doesn't seem to be the case. Nice catch @robrap!